### ISSUE TITLE
Strange eyes in GAN

### ISSUE THREAD

**Issue by xsacha** [jzijin/Super-FAN#1]
I notice in your GAN version, there are strange swirls and lines in the eyes. This doesn't happen without the GAN.

**Comment by jzijin** [comment]
> I notice in your GAN version, there are strange swirls and lines in the eyes. This doesn't happen without the GAN.
Yes.I retrained [ESRGAN](https://github.com/open-mmlab/mmsr) with face data and had the same problem.For now, I have no idea how to fix it.

**Comment by cydiachen** [comment]
> > I notice in your GAN version, there are strange swirls and lines in the eyes. This doesn't happen without the GAN.
> > Yes.I retrained [ESRGAN](https://github.com/open-mmlab/mmsr) with face data and had the same problem.For now, I have no idea how to fix it.

Dear Jzijin: 
Thank you for implementing Super_FAN. I also implemented URDGN with the same dataset settings. The URDGN is also a GAN-based FSR model. The strange swirls and lines in the eyes might be introduced by the normalization. You can refer to StyleGAN-V2. 
THX a lot for your excellent job~
